pallets/manta-xassets/src/lib.rs

```diff
@@ -0,0 +1,193 @@
+#![allow(unused_imports)]
+#![allow(dead_code)]
+#![allow(unused_variables)]
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use core::marker::PhantomData;
+use cumulus_primitives_core::ParaId;
+use frame_support::{
+	pallet_prelude::*,
+	traits::{Currency, Get, Hooks, IsType, Randomness, ReservableCurrency},
+	transactional, PalletId,
+};
+use frame_system::{
+	ensure_root, ensure_signed,
+	pallet_prelude::{BlockNumberFor, OriginFor},
+};
+use sp_runtime::{
+	traits::{Saturating, Zero},
+	DispatchResult, Permill, SaturatedConversion,
+};
+use sp_std::{vec, vec::Vec};
+use xcm::v0::{
+	Error as XcmError, ExecuteXcm, Junction, MultiAsset, MultiLocation, Order, Outcome,
+	Result as XcmResult, Xcm,
+};
+use xcm_executor::{
+	traits::{Convert, FilterAssetLocation, TransactAsset},
+	Assets,
+};
+
+pub use pallet::*;
+// Log filter
+const MANTA_XASSETS: &str = "manta-xassets";
+pub type BalanceOf<T> =
+	<<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
+
+#[frame_support::pallet]
+pub mod pallet {
+	use super::*;
+
+	#[pallet::config]
+	pub trait Config: frame_system::Config {
+		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
+		type XcmExecutor: ExecuteXcm<Self::Call>;
+		type FriendChains: Get<Vec<(MultiLocation, u128)>>;
+		type Conversion: Convert<MultiLocation, Self::AccountId>;
+		type Currency: ReservableCurrency<Self::AccountId>;
+		type SelfParaId: Get<ParaId>;
+	}
+
+	#[pallet::pallet]
+	#[pallet::generate_store(pub(super) trait Store)]
+	pub struct Pallet<T>(PhantomData<T>);
+
+	#[pallet::event]
+	#[pallet::generate_deposit(pub(super) fn deposit_event)]
+	pub enum Event<T> {
+		// PhantomData<T>,
```

Comment:
Is this commented out code forgotten or still relevant ?

Reply:
I forgot to remove it. Do it in next commit.